app/templates/events/view.hbs

```diff
@@ -89,7 +89,7 @@
                               isOpen=isEventDeleteModalOpen
                               eventName=model.name
                               deleteEvent=(action 'deleteEvent')}}
-  <div class="ui {{if isLoading 'loading'}} segment">
+  <div class="ui {{if isLoading 'loading'}} basic segment">
```

Comment:
@CosmicCoder96 I think this container should be unchanged , As I tried it , Without proper segmented boundaries all section of event are not looking nice